dataservice/api/participant/models.py

```diff
@@ -27,3 +29,39 @@ class Participant(db.Model, Base):
     demographic = db.relationship(Demographic, backref='participant',
                                   uselist=False, cascade="all, delete-orphan",
                                   lazy=True)
+
+    def upstream_immediate_relatives(self):
```

Comment:
Is this any different from `incoming_family_relationships`?

Reply:
Yes, an `incoming_family_relationship` is a FamilyRelationship obj representing an incoming edge to the participant (self). The upstream immediate relative is the Participant obj or starting node on the edge